extract/extract/extractors/amazon.py

```diff
@@ -223,14 +227,21 @@ def _get_brand(soup: BeautifulSoup, language: str) -> Optional[str]:
             "info": ("Besuche den ", "-Store", "Marke: "),
             "table": ("Marke", "Hersteller"),
         },
+        "fr": {
+            "info": ("Visiter la boutique ", "Marque\xa0: "),
+            "table": ("Marque", "Fabricant"),
+        },
     }
 
     def parse_brand(brand: str) -> Optional[str]:
         info_locales = _LANGUAGE_LOCALES[language]["info"]
         if brand.startswith(info_locales[0]):
-            return brand[len(info_locales[0]) : -len(info_locales[1])]  # noqa
-        if brand.startswith(info_locales[2]):
-            return brand[len(info_locales[2]) :]  # noqa
+            if len(info_locales) == 3:
```

Comment:
Don't use magic numbers (`3`). Define a constant with a proper name, calculate it somehow (I guess here it's the length of an expected string?), or at least comment it.